cmd/kubeadm/app/constants/constants.go

```diff
@@ -16,11 +16,14 @@ limitations under the License.
 
 package constants
 
-import "time"
+import (
+	"path"
+	"time"
+)
 
 const (
-	AuthorizationPolicyFile        = "abac_policy.json"
-	AuthorizationWebhookConfigFile = "webhook_authz.conf"
+	// KubernetesDir is the directory kubernetes owns for storing various configuration files
+	KubernetesDir = "/etc/kubernetes"
```

Comment:
It's a fine line between overridability and sprawl. Regarding this one, we decided in https://github.com/kubernetes/kubeadm/pull/156 that `/etc/kubernetes` should be a constant -- it doesn't make sense to override that really.
Also, this isn't new for this PR. If you have very strong opinions -- please file an issue and we'll see but it's not the main thing in this PR